fbw-common/src/systems/instruments/src/EFB/ToolBar/ToolBar.tsx

```diff
@@ -58,7 +59,7 @@ export const ToolBar = () => (
             <ToolBarButton to="/atc" tooltipText={t('AirTrafficControl.Title')}>
                 <BroadcastPin size={35} />
             </ToolBarButton>
-            <ToolBarButton to="/failures" tooltipText={t('Failures.Title')}>
+            <ToolBarButton to="/failures" tooltipText={t('Failures.Title')} disabled={window.FBW_REMOTE}>
```

Comment:
See above point about having an abstracted util function (that reads from ``window.FBW_REMOTE``). We don't need awareness of ``window`` at this level.